### PR TITLE
Initial spec outline

### PR DESCRIPTION
**Pull request by littledan**
This includes a complete grammar, but is just an outline of module
processing and HTML integration. Follow-on PRs would make things
more concrete.

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+          1. Let _argRef_ be the result of evaluating |AssignmentExpression|.
+          1. Let _specifier_ be ? GetValue(_argRef_).
+          1. Let _promiseCapability_ be ! NewPromiseCapability(%Promise%).
+          1. Let _specifierString_ be ToString(_specifier_).
+          1. IfAbruptRejectPromise(_specifierString_, _promiseCapability_).
+          1. Perform ! HostImportModuleDynamically(_referencingScriptOrModule_, _specifierString_, <ins>*undefined*,</ins> _promiseCapability_).
+          1. Return _promiseCapability_.[[Promise]].
+        </emu-alg>
+
+        <ins><emu-grammar>ImportCall : `import` `(` AssignmentExpression, AssignmentExpression `)`</emu-grammar></ins>
```
Comment (ljharb) [inline spec.html]: `import()` failed to allow a trailing comma in its initial version, but perhaps we could include that if we're making changes?

Comment (littledan) [reply spec.html]: Oh, sure, makes sense to me. Will update.

Comment (littledan) [reply spec.html]: Done

**Comment by MylesBorins** [comment]
I'm going to go ahead and land this so we can reference it in the explainer. Can update with another PR if there are issues

**Pull request merged**